src/main/java/frc/robot/subsystems/hood/HoodIO.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.subsystems.hood;
+
+import static edu.wpi.first.units.Units.*;
+
+import edu.wpi.first.units.*;
+import edu.wpi.first.wpilibj2.command.Command;
+import org.littletonrobotics.junction.AutoLog;
+
+import java.util.function.Supplier;
+
+public interface HoodIO {
+    HoodInputsAutoLogged inputs = new HoodInputsAutoLogged();
+
+    void updateInternalEncoder();
+
+    void resetAbsoluteEncoder();
+
+    void setAngle(MutableMeasure<Angle> angle);
+
+    void setPower(double power);
+
+    boolean atSetpoint();
```

Comment:
Same as the shoooter, this should be in the subsystem class.

Reply:
37c06c3